frontend/containers/dashboard/search-and-info/component.tsx

```diff
@@ -40,7 +40,8 @@ export const SearchAndInfo: FC<SearchAndInfoProps> = ({
 
     router.push({
       query: {
-        ...queryParams,
+        // Endpoints don't support filtering, sorting
```

Comment:
Mais na frente isto vai mudar ou não irão suportar essas funcionalidades?

Reply:
It's still unclear. It turns out the functionality is trickier than expected to implement [in the backend](https://vizzuality.atlassian.net/browse/LET-581?focusedCommentId=17296).